install-deps.sh

```diff
@@ -27,6 +27,11 @@ function in_jenkins() {
     test -n "$JENKINS_HOME"
 }
 
+[ $WITHOUT_MGR ] && with_mgr=false || with_mgr=true
```

Comment:
Why this?

Reply:
In case you are building ceph and don't want to install and build the mgr